.github/workflows/build-global-renkulab-images.yml

```diff
@@ -0,0 +1,53 @@
+name: Build global base images
+
+on:
+  push:
+    paths:
+      - "global-images/**"
+      - ".github/workflows/build-global-renkulab-images.yml"
```

Comment:
issue: since we tag the images with "latest" every time this runs lets run this on pushes to master only, in addition to the restrictions you already have for code changes.